bitwarden_license/src/Commercial.Core/SecretManagerFeatures/Projects/DeleteProjectCommand.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Bit.Core.Exceptions;
+using Bit.Core.Repositories;
+using Bit.Core.SecretManagerFeatures.Projects.Interfaces;
+
+namespace Bit.Commercial.Core.SecretManagerFeatures.Projects;
+
+public class DeleteProjectCommand : IDeleteProjectCommand
+{
+    private readonly IProjectRepository _projectRepository;
+
+    public DeleteProjectCommand(IProjectRepository projectRepository)
+    {
+        _projectRepository = projectRepository;
+    }
+
+    public async Task<List<Tuple<Guid, string>>> DeleteProjects(List<Guid> ids)
```

Comment:
I think this is a fine use case for Tuples. A curiosity question though; will we ever want to return a list of the deleted projects instead of just their ids / names? (So, instead return a list of the delete projects) The only use case I can think of for that is when we might want more info from the model, but if you are deleting you may not care anyways. (And the client can get the needed info before deletion if necessary.) Still, worth asking I think.

Reply:
I based these bulk actions off of pre-existing ones. 

Namely:
 
https://github.com/bitwarden/server/blob/40be96fc04f35718459dc9193fa36a292cd45632/src/Api/Controllers/OrganizationUsersController.cs#L360-L374

https://github.com/bitwarden/server/blob/40be96fc04f35718459dc9193fa36a292cd45632/src/Api/Controllers/ProviderUsersController.cs#L185-L198

In their services, a return tuple has the object in it.
So I guess it makes sense to follow this pattern. 

I've updated the `SecreteDeleteCommand` and `ProjectDeleteCommand` to return tuples with their respective objects. I believe this addresses the concerns brought up.

Reply:
Looks good, thanks